tools/generate.cmd

```diff
@@ -6,6 +6,22 @@
 @echo off
 setlocal
 
+:: help requested?
+set pot_help="%1%2"
+if not x%pot_help%==x%pot_help:help=% (set req_help=T)
+if not x%pot_help%==x%pot_help:?=%    (set req_help=T)
+if not "%req_help%" == "" (
```

Comment:
@jaredmoo you assume correctly, it's actually anything containing `?` or `help` (I'll slightly adjust it, `%1` is the RP name which could at some point contain `help`, who knows... on the other hand, I wanted the help to be shown if you go `/help` or `-help` or `--help`, so I gotta find some middle ground)
@shahabhijeet will do